components/pd_client/src/client.rs

```diff
@@ -571,6 +573,59 @@ impl PdClient for RpcClient {
         Ok(resp)
     }
 
+    fn get_tso(&self) -> PdFuture<TimeStamp> {
+        let timer = Instant::now();
+
+        let mut req = pdpb::TsoRequest::default();
+        req.set_count(1);
+        req.set_header(self.header());
+        let executor = move |client: &RwLock<Inner>, req: pdpb::TsoRequest| {
+            let cli = client.read().unwrap();
+            let (req_sink, resp_stream) = cli.client_stub.tso().unwrap();
```

Comment:
Establishing a stream for each request is not efficient if tso is requested very frequently. It's enough for our use now, but I think maybe we should consider implementing a more efficient tso client like the golang version in the future.

Reply:
It would be complex in the current PD client implementation, I will leave a TODO instead.